### PR TITLE
Edited README.md

### PR DESCRIPTION
**Pull request by Julian23000**
Explained what I did in the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+## Authors
+
+* Hongqian Li - Initial work - [HongqianLi](https://github.com/hongqian-li)
+* Richard Švajdlenka - Main functionality - [Richard](https://github.com/M3mberTM)
+* Julian Hack - Researcher and debugger - [JulianHack](https://github.com/Julian23000)
+* Attila Vegh - Backend Development - [AttilaVegh](https://github.com/Attila22000) pr
+
+###Julian23000
+
+I have done contributions to the project as followed. Firstly I have done research on how the ideas we had should be coded, this I have forwarded to Richard, due to him being the main coder. During the prozessione I helped debugging and testing the robot.
```
Comment (hongqian-li) [inline README.md:14]: "prozessione" there's a typo needs to be changed.

**Review by hongqian-li — CHANGES_REQUESTED** [review]
All looks great.

**Review by M3mberTM — CHANGES_REQUESTED** [review]
You have made a typo in your project. You also accidentally wrote pr after Attila's contributions so remove that

**Review by hongqian-li — CHANGES_REQUESTED** [review]
Now everything looks good. Thanks for your editing.

**Review by M3mberTM — APPROVED** [review]
Thanks for fixing the typos. Keep up the work!

**Review by hongqian-li — APPROVED** [review]

**Pull request merged**